client/src/App.js

```diff
@@ -13,7 +13,7 @@ import Settings from './components/pages/Settings';
 
 class App extends React.Component {
   state = {
-    isLogged: false,
+    isLogged: true,
```

Comment:
why changing this? default should be false

Reply:
Because I was working at the main feed page without being logged in